cmd/porter/bundle.go

```diff
@@ -103,6 +103,8 @@ For example, the 'debug' driver may be specified, which simply logs the info giv
 		"Use a bundle in an OCI registry specified by the given tag")
 	f.BoolVar(&opts.InsecureRegistry, "insecure-registry", false,
 		"Don't require TLS for the registry")
+	f.BoolVar(&opts.Force, "no-cache", false,
```

Comment:
On the build command we were going to call this flag `--force`:

https://github.com/deislabs/porter/issues/690#issuecomment-538395784

How about we use the same flag for all our commands and go with `--force`? I liked that it's not exactly docker's command because it gives us room in the future to do more without people making assumptions that the flag will work the same way as docker's.

What do you think?

Reply:
👍 `--force` it is!